docs/src/languages/svelte.md

```diff
@@ -2,16 +2,35 @@
 
 Svelte support is available through the [Svelte extension](https://github.com/zed-industries/zed/tree/main/extensions/svelte).
 
-- Tree Sitter: [Himujjal/tree-sitter-svelte](https://github.com/Himujjal/tree-sitter-svelte)
+- Tree Sitter: [tree-sitter-grammars/tree-sitter-svelte](https://github.com/tree-sitter-grammars/tree-sitter-svelte)
 - Language Server: [sveltejs/language-tools](https://github.com/sveltejs/language-tools)
 
 <!--
 TBD: Rewrite Svelte docs so it doesn't begin with a json block assuming you know what inlayHints are.
 -->
 
+## Extra theme styling configuration
+
+You can modify how certain styles such as directives and modifiers appear in attributes:
+
+```jsonc
+// Your theme's style.syntax
+"syntax": {
+  // styling for directives (eg: class:foo or on:click) (the `on` or `class` part of the attribute)
+  "attribute.function": {
+    "color": "#ff0000"
+  },
+  // styling for modifiers at the end of attributes, eg: `on:click|preventDefault|stopPropagation`
+  "attribute.special": {
+    "color": "#00ff00"
+  }
+}
+```
+
 ## Inlay Hints
 
-Zed sets the following initialization options for inlay Hints:
+Zed sets the following initialization options for inlay hints:
+
 
```

Comment:
```suggestion
Zed sets the following initialization options for inlay hints:

```

Hey, not entirely certain, but I believe Prettier dislikes this added newline (because there is already one below) and thus fails.